app/scripts/controllers/loanAccount/ViewLoanDetailsController.js

```diff
@@ -596,6 +596,14 @@
               }
               return true;
             };
+
+            scope.checkStatus = function(){
```

Comment:
would be better to have a name which describes the condition being checked

Ex: validationLoanHasCrossedActivation